cli/pom.xml

```diff
@@ -75,6 +75,21 @@
       <artifactId>progressbar</artifactId>
       <version>0.10.0</version>
     </dependency>
+    <dependency>
+      <groupId>org.jline</groupId>
+      <artifactId>jline</artifactId>
+      <version>3.23.0</version>
+    </dependency>
+    <dependency>
+      <groupId>org.fusesource.jansi</groupId>
+      <artifactId>jansi</artifactId>
+      <version>2.4.0</version>
+    </dependency>
+    <dependency>
+      <groupId>org.jline</groupId>
+      <artifactId>jline-terminal-jansi</artifactId>
+      <version>3.23.0</version>
+    </dependency>
```

Comment:
Great stuff with this awesome libs that give us amazing features.
Can you group the dependencies per `groupId` and use a variable `${jline.version}` to avoid redundancy and simplify maintenance (upgrades)?

Reply:
Removed.